### ISSUE TITLE
Linux versus Windows

### ISSUE THREAD

**Issue by ppdewolf** [julienjamme/manual_tau_argus_to_quarto#1]
Minor detail:
the script `from_md_toquarto_book.R` works (probably) for Linux, but not for Windows.
For Windows you have to replace some `\n` by `\r\n`

**Comment by julienjamme** [comment]
Yes, I've worked on linux platform. I'll add the alternative for Windows platform in the code.
Thanks.
